ceph/ceph_admin/jinja_templates/snmp_destination.jinja

```diff
@@ -1,3 +1,9 @@
+{% if "snmp_v3_auth_username" in spec["spec"]["credentials"] %}
```

Comment:
Add check for both username and password

Reply:
ack.
done